network-model/src/commonMain/kotlin/com/wire/kalium/network/api/authenticated/featureConfigs/FeatureConfigResponse.kt

```diff
@@ -163,7 +163,7 @@ sealed class FeatureConfigData {
         @SerialName("status")
         val status: FeatureFlagStatusDTO,
         @SerialName("config")
-        val config: ConferenceCallingConfigDTO
+        val config: ConferenceCallingConfigDTO? // TODO make it non optional when the backend is ready
```

Comment:
Instead of when the backend ready, it should be when the minimum supported API is v6